src/index.html

```diff
@@ -2,34 +2,19 @@
 <html>
 	<head>
 		<title>Angular 2 Grid Demo</title>
-		<link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.min.css" />
-		<script src="node_modules/es6-shim/es6-shim.min.js"></script>
-		<script src="node_modules/zone.js/dist/zone.js"></script>
-		<script src="node_modules/reflect-metadata/Reflect.js"></script>
-		<script src="node_modules/systemjs/dist/system.js"></script>
+		<link rel="stylesheet" href="/node_modules/bootstrap/dist/css/bootstrap.min.css" />
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
```

Comment:
Bootstrap is included in the bower dependencies. There's no harm moving to npm as none of the other deps are in bower. However, it's just the CSS imported from bootstrap. None of the js is used so jquery isn't required :)